src/config.rs

```diff
@@ -113,7 +118,7 @@ impl Context {
     pub fn set_config(&self, key: Config, value: Option<&str>) -> Result<(), Error> {
         match key {
             Config::Selfavatar if value.is_some() => {
-                let rel_path = std::fs::canonicalize(value.unwrap())?;
+                let rel_path = std::fs::canonicalize(value.unwrap_or_default())?;
```

Comment:
line above says `value.is_some()` so this can't fail.

Reply:
while that's true it also means that the "or_default" part is guaranteed not invoked ;) 
i think @r10s's point here is that if somebody changes the "guarding" code for whatever reason that unwrap_or_default() will still prevent a crash.

Reply:
yip that was my intention. but i am fine using a "unwrap_or_panic()" if the check is obvious, just a few lines before.